build.gradle

```diff
@@ -1,114 +1,147 @@
 /// Gradle Build script for 18xx Rails program (rails.sf.net)
 
-import org.apache.tools.ant.filters.*
 import java.text.SimpleDateFormat
 
-
-
-// Addition of JavaProfile plugin to read Java property files
-// reference: https://github.com/unsaved/gradle-javaPropFile-plugin
-
-buildscript {
-    repositories { jcenter() }  // Before plugin v. 1.0.0 use mavenCentral()
-    dependencies {
-        classpath 'com.admc:gradle-javaPropFile-plugin:1.0.0'
-        // For strictly repeatable builds, replace 'latest.integration'
-        // with the latest literal GA version label.
-    }
-}
-
-
-// Addition of launch4j gradle plugin
-// refer to https://github.com/TheBoegl/gradle-launch4j
-
-buildscript {
-  repositories {
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
-  }
-  dependencies {
-    classpath "gradle.plugin.edu.sc.seis.gradle:launch4j:2.3.0"
-  }
-}
-
-// Addition of shadow plugin to create one jars
-// reference: https://github.com/johnrengelman/shadow
-plugins {
-  id 'java' // or 'groovy' Must be explicitly applied
-  id 'com.github.johnrengelman.shadow' version '5.0.0'
-}
-
-// Addition of macAppBundle
-// https://github.com/crotwell/gradle-macappbundle
 plugins {
+  id 'java'
+  id 'maven-publish'
+  // https://github.com/johnrengelman/shadow
+  id 'com.github.johnrengelman.shadow' version '5.2.0'
+  // plugin to check for dependency updates via task dependencyUpdates
+  // https://github.com/ben-manes/gradle-versions-plugin
+  id 'com.github.ben-manes.versions' version '0.28.0'
```

Comment:
I would recommend to use services like [dependabot](https://dependabot.com/) instead

Reply:
Worth looking into but it seems dependabot's focus is on automating dependency updates vs this reporting on it? It would seem that we have a place for both of these?

Reply:
When a new dependency version is released dependabot opens a new PR with the necessary update. Afterwards the maintainers of the project have the choice to either merge the PR or reject it. I think it is quite a good approach.